api/v1alpha1/database_webhook.go

```diff
@@ -109,82 +115,93 @@ func (r *Database) Default() {
 
 var _ webhook.Validator = &Database{}
 
-func ndbServerSpecValidatorForCreate(r *Database, allErrs field.ErrorList, ndbPath *field.Path) field.ErrorList {
+func ndbServerSpecValidatorForCreate(ndb *NDB, allErrs field.ErrorList, ndbPath *field.Path) field.ErrorList {
 	databaselog.Info("Entering ndbServerSpecValidatorForCreate...")
 
-	if r.Spec.NDB == (NDB{}) {
-		allErrs = append(allErrs, field.Invalid(ndbPath, r.Spec.NDB, "NDB spec field must not be empty"))
+	if ndb == &(NDB{}) {
+		allErrs = append(allErrs, field.Invalid(ndbPath, ndb, "NDB spec field must not be empty"))
```

Comment:
I don't think we'll ever hit this case, because: the address of `(NDB{})` will not be known to the user. Would it be better to check `*ndb == (NDB{})` instead?